.github/workflows/build-deploy.yml

```
@@ -5,6 +5,8 @@ on:
   push:
     branches:
       - ep2025
+  schedule:
+    - cron: "*/10 * * * *"
```

```suggestion
    - cron: "0 * * * *"
```

I believe hourly builds would be enough for now.